### PR TITLE
New UIUC team with fixed bugs and compressed images.

### PR DESCRIPTION
**Pull request by davinwid**
Compressed images to make sure page loads faster. 
Fixed line break bug for members with multiple positions.

### CODE REVIEW

File: uiuc-team.html [hunk 1]
```diff
@@ -72,7 +72,7 @@ <h2><strong>Our Mission</strong></h2>
                     <div class="content">
                         <section>
                             <a class="image featured"><img src="images/team_new.jpg" alt="" /></a>
-                            <header>
+                            <header class>
```
Comment (linahsie) [inline uiuc-team.html]: ini  'class' di delete ya

File: assets/sass/main.css [hunk 2]
```diff
@@ -2223,3 +2222,6 @@ table, th, td {
 .position-text {
   margin-bottom: 5.5%;}
 
+span.member-title {
```
Comment (linahsie) [inline assets/sass/main.css]: buat css code, ini kan ada hubungannya sm team-member section, taro css code nya di bawah relevant code (as in like taro ini dibawah .team-member {} to make it easier if we want to edit it again)

File: assets/sass/main.css [hunk 3]
```diff
@@ -1753,8 +1753,7 @@ body.index #main {
     padding: 3em; } }
 
 /* Narrower */
-#navPanel, #navButton {
-  display: none; }
```
Comment (linahsie) [inline assets/sass/main.css]: put this back ya!

File: assets/sass/main.css [hunk 4]
```diff
@@ -2223,3 +2222,6 @@ table, th, td {
 .position-text {
   margin-bottom: 5.5%;}
 
+span.member-title {
+  display: block;
```
Comment (linahsie) [inline assets/sass/main.css]: and change this to--- display: block !important;
we don't want to change code that we didn't write (yang #navPanel), so we add !important to force the styling that we want.

**Review by linahsie — CHANGES_REQUESTED** [review]
Some small changes. Yaaaay good job Vin! 💃

**Comment by davinwid** [comment]
Danke schön Lina senpai

File: assets/sass/main.css [hunk 5]
```diff
@@ -2222,6 +2223,6 @@ table, th, td {
 .position-text {
   margin-bottom: 5.5%;}
 
-span.member-title {
-  display: block;
+.team-member-title {
```
Comment (linahsie) [inline assets/sass/main.css]: move this line to below line 2089

**Review by linahsie — CHANGES_REQUESTED** [review]

File: assets/sass/main.css (hunk 5 above)
Comment (linahsie) [reply assets/sass/main.css]: maksudnya: css styling buat team member di kumpulin semuanya, jadi ga susah nyari nya

**Review by linahsie — APPROVED** [review]

**Pull request merged**